docs/metrics.md

```diff
@@ -4,7 +4,6 @@
 This document enumerates the metrics collected by this project.
 This project may depend on other projects which also collect metrics.
 This means you might have to go searching through the dependency tree to get a full picture of everything collected by this project.
-Sorry about that.
```

Comment:
Glean's automagical docs thing did this automagically; I guess they're not sorry any more 😆